buildSrc/src/main/resources/templates/pojo.mustache

```diff
@@ -86,6 +86,14 @@ public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}} {{#seriali
 
     return this;
   }
+
+  public {{items.datatypeWithEnum}} get{{nameInCamelCase}}Value({{{items.datatypeWithEnum}}} {{name}}Key) {
```

Comment:
Since this will be recognized as a getter, will this show up as an additional field to the serialized DTOs?

Reply:
This method will be generated for all auto-generated DTOs.

Reply:
I don't think it will be recognized as a getter, since it has accepts an argument

Reply:
hmm, I'll revert the template changes rather create a new method inside `ConsumerResource.java`. Template changes could difficult to keep track on. We'll have to update this in `OpenAPIGenerator`.